ceph/ceph_admin/add.py

```diff
@@ -0,0 +1,48 @@
+from typing import Dict, Union
+
+from ceph.utils import get_node_by_id
+
+from .common import config_dict_to_string
+from .typing_ import DaemonProtocol, OrchProtocol
+
+
+class AddMixin:
+    def add(self: Union[OrchProtocol, DaemonProtocol], config: Dict):
+        """
+
+
+        Execute the add method using the object's service name.
+        Args:
+            config:     Key/value pairs passed from the test suite.
+                        pos_args        - List to be added as positional params
+
+        Example:
+            config:
+              service: osd
+              command: add
+              base_cmd_args:
+                verbose: true
+              pos_orgs:
+                - node1
+                - /dev/vdb
+        """
+        service = config.pop("service")
+        base_cmd = ["ceph", "orch", "daemon", "add", service]
+        if config.get("base_cmd_args"):
```

Comment:
Since orch commands support positional args which could lead to issue when base-cmd-args added after `daemon add`. please make sure it work fine.

Reply:
Hoping that command works fine without changing code or its arrangement.